<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionShake"/>
    <addaction name="actionOrder"/>
   </widget>
   <widget class="QMenu" name="menuGraphic_settings">
    <property name="title">
     <string>Graphic Settings</string>
    </property>
    <addaction name="actionLoadImage"/>
    <addaction name="actionStandartBlocks"/>
    <addaction name="actionLoadBackground"/>
    <addaction name="actionStandartBackground"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuActions"/>
   <addaction name="menuGraphic_settings"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionShake">
   <property name="text">
    <string>Shake</string>
   </property>
  </action>
  <action name="actionLoadImage">
   <property name="text">
    <string>Load Image</string>
   </property>
  </action>
  <action name="actionOrder">
   <property name="text">
    <string>Order</string>
   </property>
  </action>
  <action name="actionStandartBlocks">
   <property name="text">
    <string>Standart Blocks</string>
   </property>
  </action>
  <action name="actionLoadBackground">
   <property name="text">
    <string>Load Background</string>
   </property>
  </action>
  <action name="actionStandartBackground">
   <property name="text">
    <string>Standart Background</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
